docker/daiquiri/rootfs/home/dq/sh/install-app.sh

```diff
@@ -1,31 +1,34 @@
 #!/bin/bash
 
-
 source "${HOME}/.bashrc"
+source "${HOME}/.venv/bin/activate"
+
+reqfile="${DQAPP}/pyproject.toml"
+echo "current directory: $(pwd)"
 
-reqfile="${DQAPP}/requirements.txt"
+cd "${DQAPP}"
 
 if [[ -f "${reqfile}" ]]; then
+    
     # if the path to the local daiquiri repo exists then don't install it from the req
     if [[ -d "${DQSOURCE}" ]]; then
         echo "Install app requirements excluding daiquiri"
-        pip install $(grep -v "^\s*#" ${reqfile} | grep -ivE "django-daiquiri")
+        uv pip install -e .
     else
         echo "Install app requirements"
-        pip install -r ${reqfile}
+        uv pip install --extra daiquiri -e .
```

Comment:
It seems that `pyproject.toml` must be explicitly provided here. I think that it's related to https://github.com/astral-sh/uv/issues/14558 .

The following worked for me
```bash
uv pip install -r ${reqfile} --extra daiquiri -e .
```